.env.example

```diff
@@ -105,8 +105,9 @@ GITHUB_TOKEN=
 ## Secret storage
 # SECRET_STORAGE_BACKEND= # optional, should be one of: SecretStorage::DbBackend (default) or SecretStorage::HashicorpVault
 
-## Kubernets sidecar
+## Kubernetes
 # SECRET_SIDECAR_IMAGE= # optional, path to the docker image that built from the secret_puller service.  service will be disabled w/o it
+# KUBERNETES_IMAGE_PULL_SECRETS="ecr-auth-us-west-1,..." # override imagePullSecrets before deploying to make docker authenticate to your repository
```

Comment:
Suggestion: include a URL for http://kubernetes.io/docs/user-guide/images/#specifying-imagepullsecrets-on-a-pod

Comment that this is a comma-separated list of the names of secrets that are expected to be present in the cluster/namespace that you're deploying to.